src/gui/src/components/explorer-grid/selected-item/tabs-overview.tsx

```diff
@@ -133,8 +205,24 @@ export const OverviewTabContent = () => {
         </div>
       }
 
+      {manifest?.contributors && manifest.contributors.length > 0 && (
```

Comment:
Like mentioned above, let's get a `contributors` property in the context store instead of retrieving it directly from the manifest.